Apps/Common/MultiPatchModelGenerator.C

```diff
@@ -848,6 +859,22 @@ bool MultiPatchModelGenerator3D::createTopologySets (SIMinput& sim) const
       for (size_t i = 0; i < nx; ++i)
         insertion(TopItem(IJK2K(i,j,k),r,2), "Boundary", "Face");
 
+  TopEntity& bx  = sim.topology("BoundaryX");
+  TopEntity& by  = sim.topology("BoundaryY");
+  TopEntity& bz  = sim.topology("BoundaryZ");
+
+  auto side_insertion = [&sim](TopEntity& set, int start)
```

Comment:
All the other lambda's in this file are declared `auto&&`, so why not here as well for the consistency? (don't remember if there is a practical difference in this case or not).

Reply:
changed.